src/Components/List/List.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import '../List/List.css';
+
+export default function List({ items }) {
+  let history = useHistory();
+
+  const redirectPath = () => {
+    history.push('/add-item');
+  };
+
+  return (
+    <div className="List">
+      {items.length === 0 ? (
+        <section className="emptyList">
+          <p>Your shopping list is empty. Add a new item to start your list.</p>
+          <button onClick={redirectPath}>Add New Item</button>
+        </section>
+      ) : (
+        <React.Fragment>
+          <h3>Item List:</h3>
+
+          <ul>
+            {items.map((item) => (
+              <li key={item.name}>{item.name}</li>
+            ))}
+          </ul>
```

Comment:
Nice use of Fragments here to render multiple elements! There's also a shortcut for these where you can just use empty angle brackets.
```
<>
    <div></div>
    <div></div>
</>
```

Reply:
Wow this is super cool, thanks for sharing! Eden and I will leave the React fragment as is for now, for readability on our end 😅